docs/en/latest/config.json

```
@@ -295,4 +302,4 @@
       "href": "https://github.com/apache/apisix/blob/master/CHANGELOG.md"
     }
   ]
-}
+}
```

```suggestion
}

```
Need a blank line at the end of file.